apps/condo/lang/en/en.json

```diff
@@ -70,6 +70,8 @@
   "example.Email": "name@example.org",
   "example.Name": "Jon Johnson",
   "example.Phone": "+1 (300) 300 22 22",
+  "HotReload.title": "System update!",
+  "HotReload.message": "We have updated the system, so this page will now reload",
```

Comment:
```suggestion
  "HotReload.message": "We have updated the system, this page will now be reloaded",
```

Reply:
Will be removed after conversation with Max